src/hooks/getBookbyId.tsx

```diff
@@ -0,0 +1,20 @@
+import { useState, useCallback, useEffect } from "react";
+import { getBookInfo } from "../api/books";
+import { Book } from "../api/types/book";
```

Comment:
use absolute paths for imports because they are cleaner than relative imports. Search how to do it in create react app for a typescript project. You'll probably need to change the `baseUrl` in `tsconfig.json`